js/ui/grid_core/ui.grid_core.editing.js

```diff
@@ -1698,7 +1730,7 @@ const EditingController = modules.ViewController.inherit((function() {
 
                 if(isError) {
                     if(editData) {
-                        editData.error = arg;
+                        this._getInternalData(editData.key, true).error = arg;
```

Comment:
Add addInternalData method to use here it instead of _getInternalData. Also the second boolean parameter is not used in _getInternalData.
It is not good that _getInternalData method uses for internal data creating.